### ISSUE TITLE
black

### ISSUE THREAD

**Issue by tonyakdewey** [lingdb/ielex2#20]
The description says 'the natural colour of charcoal', but the example sentence is "The sky was moonless and black".

In Old Norse these would be two separate words.  Charcoal is svartr, but the night sky is blá (yes, I know that word comes to mean 'blue' in the modern language, but in the older language it clearly means 'black', since it is used to describe hair color, the skin color of North Africans, horses, rotten teeth, and clothing worn by those about to die).  Old English has a similar problem for blæc and sweart.

Can we change the example sentence to something not with the night sky (which in any case is problematic, since some people would call that 'dark blue')?

Also, can we further specify the description so I can exclude one of my 'black' words for the older Germanic languages?

**Comment by leszekjocz** [comment]
I think that 'sky' is a bad idea because it will elicit 'dark' and not 'black' in many languages.

**Comment by cormacanderson** [comment]
All sounds reasonable. What should we use as the example then? Which of your black words do you think we should go for @tonyakdewey ?

**Comment by tonyakdewey** [comment]
For the older Germanic languages it's really really hard to say which is primary, and the modern languages tend to preserve either one or the other, so I think both should stay in for the older languages.  In the older languages they don't so much overlap as make a distinction that the modern languages don't make, which is why I don't want to claim one is primary over the other.  But depending what example we use will make a difference as to what I mark as the Swadesh term.

If we say "Charcoal is black", then I would mark svartr as the Swadesh term for Old Norse.  If we say "Ravens are black", I would mark blá as the Swadesh term.  But both of these are good examples of "black" in English (and I would call them both "schwarz" in German).  "Ravens are as black as charcoal" would just make my head explode.  =-)

I don't really care which one we pick, since I'm going to leave both in the database, but we should all be in agreement, so that I'm not using charcoal-black and @findell raven-black, or vice versa.

**Comment by findell** [comment]
I agree with the comments on the night sky (even if I rather like it aesthetically).  Charcoal might be the best example (unless it's going to be confused with "grey" or something like that).
